src/output-json.c

```diff
@@ -456,6 +472,12 @@ OutputCtx *OutputJsonInitCtx(ConfNode *conf)
             }
         }
 
+        const char *prefix = ConfNodeLookupChildValue(conf, "prefix");
+        if (prefix != NULL)
+        {
+            json_ctx->file_ctx->prefix = SCStrdup(prefix);
```

Comment:
This is going to fail our qa (coccinelle check) because it's a memory allocation that isn't checked. It shouldn't lead to problems in this specific case, but I need a fix anyway :)

Also, I don't see where this memory is freed.